doc/interfaces/data_fetch.rst

```diff
@@ -0,0 +1,23 @@
+-------------------------
+Downloading DIPY datasets
+-------------------------
+In this tutorial, you will learn how to download the DIPY datasets using the terminal.
+
+First, let's create a folder::
+
+    mkdir data_folder
+
+You can list all available datasets in DIPY using the command::
+
+    dipy_fetch list
+
+In order to fetch a specific dataset, you will specify its name with the ``dipy_fetch``
+command, and an optional destination directory using the ``--out_dir`` argument::
+
+    dipy_fetch {specific_dataset} --out_dir {data_folder}
+
+For example, to download the ``sherbrooke_3shell`` dataset, you will run::
```

Comment:
For example, to download the {double_backticks}sherbrooke_3shell{double_backticks} dataset, you would run::

Reply:
Reworded in 8899fa6